packages/gatsby-plugin-sharp/src/scheduler.js

```diff
@@ -152,6 +155,7 @@ function runJobs(
     )
 
     Promise.all(promises).then(() => {
+      console.log(`[gatsby-plugin-sharp] finishing job ${job.inputPath}`)
```

Comment:
changes here are just for debugging purpose - they will need to be removed later

Reply:
Is there not an verbose/debug mode for Gatsby to show all this debug messages ?

Reply:
there is, but even then - this would be way to spammy